### PR TITLE
replace env with parameters

### PR DESCRIPTION
**Pull request by shumyachkin**

### CODE REVIEW

**Review by mussbach — CHANGES_REQUESTED** [review]
please use lowersnailcase as for all other params. And define a defaultvalue (empty) in parameters_default.yml

File: demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml [hunk 1]
```diff
@@ -325,6 +325,15 @@ parameters:
     saml_sp_x509_string: ''
     saml_registration_url: ''
 
+    # knpu_oauth2_client settings
```
Comment (mussbach) [inline demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml:328]: You need to define oauth_keycloak_client_id, oauth_keycloak_client_secret, oauth_keycloak_auth_server_url and oauth_keycloak_realm, not the ones defined here

**Review by mussbach — CHANGES_REQUESTED** [review]

**Review by graupnerdemos — APPROVED** [review]
This looks correct. However, I think this should go to `main` now?

**Comment by mussbach** [comment]
No, release is fine as we want to test it with the Keycloak in the other environment

**Review by mussbach — APPROVED** [review]
lgtm, thx

**Pull request merged**